microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -39,6 +39,14 @@ This release adds improvements related to the following components and concepts.
 //[id="microshift-4-14-new-feat-based-on-{op-system-ostree}"]
 //==== Placeholder for new feat bases on RHEL Edge
 
+[id="microshift-backup-and-restore"]
+=== Backup and restore
+The capability to back up and restore the {product-title} database is now available.
+
+* On `rpm-ostree` systems such as {op-system-ostree}, {product-title} automatically creates a backup on every start. These automatic backups are deleted and replaced with the latest backup each time the system restarts.
```

Comment:
Let's not mention this here. Those backups are really only meant for rollback support, and we don't want users messing around with them.